data/formats-data.js

```diff
@@ -5104,7 +5111,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	golett: {
-		randomBattleMoves: ["earthquake", "shadowpunch", "dynamicpunch", "icepunch", "stealthrock", "rockpolish"],
+		randomBattleMoves: ["earthquake", "shadowpunch", "drainpunch", "icepunch", "stealthrock", "thunderpunch"],
```

Comment:
dynamic punch still has its use with no guard

Reply:
but when 2/3 of my moves are damaging punches i want iron fist all the time